src/twisted/words/test/test_domish.py

```diff
@@ -181,14 +181,6 @@ def test_addContentNativeStringASCII(self):
         self.assertEqual("native", str(element))
 
 
-    def test_addContentBytes(self):
```

Comment:
Typing is considered optional for Python as a language, but I don't think it should be considered optional for Twisted as a project. Since Twisted has mypy as one of the checks in CI, developers working on Twisted cannot ignore mypy.

That said, there are two scenarios where mypy cannot detect type errors:
- when the calling code inside Twisted isn't annotated yet
- when a project using Twisted isn't using mypy

Over time, the first category should shrink to zero, but that may take a while. I think the latter category is also likely to shrink over time, but that is a theory that is unproven at this time. Right now the runtime type check still has some value, so I'll re-add it.